packages/commonwealth/client/scripts/views/components/MetaTags/MetaTags.tsx

```diff
@@ -82,7 +82,7 @@ const defaultMeta = {
   },
   'twitter:image': {
     name: 'twitter:image',
-    content: `https://${PRODUCTION_DOMAIN}/img/brand_assets/common.png`,
+    content: 'https://images.com/image.jpg',
```

Comment:
This line should actually be content:`https://${PRODUCTION_DOMAIN}/img/brand_assets/summary_large_image.png`